L1Trigger/TrackTrigger/plugins/TTStubBuilder.cc

```diff
@@ -219,9 +219,56 @@ void TTStubBuilder<Ref_Phase2TrackerDigi_>::produce(edm::Event& iEvent, const ed
           // We keep is in the stub final container tough, but flagged as reject by FE
 
           if (FEreject) {
-            tempTTStub2.setTriggerDisplacement(500 + 2. * tempTTStub.getTriggerDisplacement());
-            tempTTStub2.setTriggerOffset(500 + 2. * tempTTStub.getTriggerOffset());
-            tempTTStub2.setRealTriggerOffset(500 + 2. * tempTTStub.getRealTriggerOffset());
+            tempTTStub2.setRawBend(500 + 2. * tempTTStub.getRawBend());
+            tempTTStub2.setBendOffset(500 + 2. * tempTTStub.getBendOffset());
+
+            tempInner.push_back(*(tempTTStub2.getClusterRef(0)));
+            tempOuter.push_back(*(tempTTStub2.getClusterRef(1)));
+            tempAccepted.push_back(tempTTStub2);
+            continue;
+          }
+
+          (isPS) ? maxStubs = maxStubs_PS_CIC_5 : maxStubs = maxStubs_2S_CIC_5;
```

Comment:
this is an unusual use of a ternary operator: more readable would be
```
maxStubs = isPS ? maxStubs_PS_CIC_5 : maxStubs_2S_CIC_5;
```

Reply:
fixed